### PR TITLE
ZStream Satellite Upgrade Support

### PR DESCRIPTION
**Pull request by jyejare**
This PR adds a zStream Upgrade Support to our Upgrade Automation.

### CODE REVIEW

**Comment by jyejare** [comment]
@SatelliteQE/quality-engineers , Please review this , Its pending from long time !

**Comment by JacobCallahan** [comment]
ACK, looks sane to me.

File: automation_tools/satellite6/upgrade/__init__.py [hunk 1]
```diff
@@ -165,9 +166,14 @@ def product_upgrade(product):
...
     if check_necessary_env_variables_for_upgrade(product):
+        from_version = os.environ.get('FROM_VERSION')
+        to_version = os.environ.get('TO_VERSION')
         sat_host, cap_hosts, clients6, clients7 = setup_products_for_upgrade(
             product, os.environ.get('OS'))
-        execute(satellite6_upgrade, host=sat_host)
+        if from_version != to_version:
+            execute(satellite6_upgrade, host=sat_host)
+        elif from_version == to_version:
+            execute(satellite6_zstream_upgrade, host=sat_host)
```
Comment (sghai) [inline automation_tools/satellite6/upgrade/__init__.py]: 👍

**Comment by sghai** [comment]
ACK.. A minor nitpick.. please address this and we can merge the PR.

**Comment by jyejare** [comment]
@sghai : I worked on the review comments.
@lpramuk , Can you please review this PR ?
@SatelliteQE/quality-engineers , Review this PR please.

**Comment by sghai** [comment]
ACK

**Pull request merged**